Vote for Douxianming as a reviewer. (#759)
pingcap/community: votes/0759-dhysum-as-tidb-reviewer.md

# A Vote for BornChanger as TiDB Reviewer

## Proposal

> [@dhysum](https://github.com/dhysum) has been working on `pingcap/tidb` and design and implement DDL job pause and resume.

Reviewer: ```suggestion
[@dhysum](https://github.com/dhysum) designed and implemented the [DDL Job Pause and Resume](https://github.com/pingcap/tidb/issues/40041) feature for TiDB. He also participated in the design and development of the [Pause DDL for TiDB Upgrade](https://github.com/pingcap/tidb/issues/39751) feature.
```

> 
> * [#40041](https://github.com/pingcap/tidb/issues/40041) and related issue [#39751](https://github.com/pingcap/tidb/pull/39751)

Reviewer: ```suggestion
```

> The work has been done with great effort and it obviously improved the maintainability of the TiDB repository, all his contributions are listed here:

Reviewer: ```suggestion
The work has been done in a scalable way, his vast engineering experience is impressive. All his contributions are listed here:
```

# A Vote for dhysum as TiDB Reviewer

## Proposal

[@dhysum](https://github.com/dhysum) designed and implemented the [DDL Job Pause and Resume](https://github.com/pingcap/tidb/issues/40041) feature for TiDB. He also participated in the design and development of the [Pause DDL for TiDB Upgrade](https://github.com/pingcap/tidb/issues/39751) feature.

The work has been done in a scalable way, his vast engineering experience is impressive. All his contributions are listed here:

* [Authored commits](https://github.com/pingcap/tidb/commits?author=dhysum)

I (@Benjamin2037) hereby nominate @dhysum as TiDB Reviewer and call for a vote.

## Deadline

The vote will be open for at least 3 days unless there is an objection or not enough votes.

## Scope

* Team TiDB

## Result
Approved by 2 binding votes:

> * wjhuang2016

Reviewer 2: ```suggestion
```

* disksing
* jackysp

> * AndreMouche

Reviewer 2: ```suggestion
```
